avocado/utils/lv_utils.py

```diff
@@ -0,0 +1,517 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Author: Harish <harisrir@linux.vnet.ibm.com>
+# Copyright: 2016 IBM
+#
+# Based on the code by
+# Author: Plamen Dimitrov
+# Copyright: Intra2net AG 2012
```

Comment:
(I mean in 10 years no one would know where this file come from, whether autotest, footest or bartest)